src/runtime/manifest-parser.pegjs

```diff
@@ -1474,12 +1475,13 @@ RecipeSlot
   }
 
 SchemaInline
-  = names:((upperIdent / '*') whiteSpace?)* '{' multiLineSpace fields:(SchemaInlineField (',' multiLineSpace SchemaInlineField)*)? ','? multiLineSpace '}'
+  = names:((upperIdent / '*') whiteSpace?)* '{' multiLineSpace fields:(SchemaInlineField (',' multiLineSpace SchemaInlineField)*)? ','? multiLineSpace allFields:('*'?) multiLineSpace '}'
```

Comment:
Please just add `'*'` to the options for `SchemaInlineField` instead, otherwise this parser isn't particularly user friendly (i.e. I think it will parse `{a: Text, *}` but not {*, a: Text}`).